options.mk

```diff
@@ -187,6 +187,11 @@ ifeq ($(WOLFTPM),1)
   endif
 endif
 
+ifeq ($(MEASURED_BOOT),1)
```

Comment:
Please add this new option `MEASURED_BOOT` to the `tools/config.mk`. Add documentation for the new option. Note it requires `.config` with `WOLFTPM=1` and `MEASURED_BOOT=[PCRNUM]`.

Reply:
I was wondering if I could code this requirement the way we do in wolfTPM, but it seems with option.mk such dependencies are not possible. I will make sure to add detailed info in documentation. Thanks

Reply:
Yes, code it to force WOLFTPM=1 in the Makefile

Reply:
or even better, in `options.mk`